api/controllers/v1/entrance/find.js

```diff
@@ -9,86 +9,71 @@ const DescriptionService = require('../../../services/DescriptionService');
 const HistoryService = require('../../../services/HistoryService');
 const LocationService = require('../../../services/LocationService');
 const { toEntrance } = require('../../../services/mapping/converters');
+const ErrorService = require('../../../services/ErrorService');
 
 module.exports = async (req, res) => {
-  TEntrance.findOne(req.params.id)
-    .populate('author')
-    .populate('cave')
-    .populate('documents')
-    .populate('geology')
-    .populate('names')
-    .exec(async (err, found) => {
-      const params = {};
-      const entrance = found;
-      params.searchedItem = `Entrance of id ${req.params.id}`;
+  try {
+    const entrance = await TEntrance.findOne(req.params.id)
+      .populate('author')
+      .populate('reviewer')
+      .populate('cave')
+      .populate('documents')
+      .populate('names');
 
-      if (err) {
-        return res.serverError({
-          error: err,
-          message: `An unexpected server error occured when trying to get ${params.searchedItem}`,
-        });
-      }
-      if (!entrance) {
-        return res.notFound(`${params.searchedItem} not found`);
-      }
+    const params = { searchedItem: `Entrance of id ${req.params.id}` };
+    // TODO How to delete/restore entity ?
+    if (!entrance || entrance.isDeleted)
+      return res.notFound(`${params.searchedItem} not found`);
```

Comment:
@vmarseguerra : Je suis en train de travailler sur la page des entrance "deleted". On ne veut pas afficher un "not found" quand une entrance est "is_deleted" mais afficher des informations minimalistes sur cette entrances et potentiellement un lien vers l'entrance associé via la propriété "redirect_to".

Pour les informations minimalistes, je voyais quelque chose comme cela : 
![image](https://user-images.githubusercontent.com/985903/205522928-6d44b578-1b97-43f8-8a52-b04c9f2bb611.png)

Le rendu d'une entrance deleted serait : 

![image](https://user-images.githubusercontent.com/985903/205522985-a55959bd-ec62-459a-91c1-c5597e00d542.png)

Reply:
L'objectif de retourner une entrance minimaliste est plus tard d'avoir des actions possibles sur la page d'une entité deleted pour aider un modérateur à comprendre la suppression, potentiellement l'annuler ou récupérer certaines informations,,,

Reply:
Oui ok ça marche. Tu veux que je change quelque chose dans le PR ? Où tu préfères tout mettre dans celui des deleted ?